datasync/chaindatafetcher/kafka/consumer.go

```diff
@@ -21,10 +21,16 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"log"
 
 	"github.com/Shopify/sarama"
 )
 
+// Logger is the instance of a sarama.StdLogger interface that chaindatafetcher leaves the SDK level information.
+// By default it is set to discard all log messages via ioutil.Discard, but you can set it to redirect wherever you want.
+var Logger sarama.StdLogger = log.New(ioutil.Discard, "[Chaindatafetcher] ", log.LstdFlags)
```

Comment:
Why did you discard logs by default? For me, printing stdout or stderror looks better.

Reply:
The logger and the policy are the same as sarama package, but I agree with the stdout as default. I will update this.